onnxruntime/core/providers/cuda/tensor/gather_nd.cc

```diff
@@ -119,9 +87,6 @@ Status GatherNDBase::CommonComputeKernel(
       indices_data,
       input_slice_offsets_buffer.get());
 
-  utils::MLTypeCallDispatcher<ComputeImpl, float, MLFloat16, double>
```

Comment:
Removing this from this function, would give us an opportunaty to reuse the PrepareCompute in the CPU GatherND/GatherNDGrad kernels (I can change later)